profiles/views.py

```diff
@@ -127,3 +127,14 @@ class ViewedCompanyList(ListCreateAPIView):
     def get_queryset(self):
         user_id = self.request.user.id
         return ViewedCompany.objects.filter(user=user_id)
+
+
+class GetUserId(ListAPIView):
+    queryset = Profile.objects.all()
+    serializer_class = ProfileSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def list(self, request):
+        profile = Profile.objects.get(person_id=self.request.user)
+        serializer = ProfileSerializer(profile)
+        return Response(serializer.data)
```

Comment:
1. Why `ListAPIView`?
2. What for do we have `queryset` property here, is it used anywhere?
3. What for do we have `searializer_class` property here?
3. Why do we _still_ (after probablly 10+ mentions in other PRs) 'handcraft' reponses?

Reply:
Thank you for feedback. I have deleted it at all. I am using filtering for now